server/py/framework/utils/db/sql_types.py

```diff
@@ -29,7 +29,7 @@ class _Timestamp:
 
     class _Datetime:
         sqlite = sqlalchemy.DATETIME(timezone=True)
-        mysql = sqlalchemy.dialects.mysql.DATETIME(timezone=True, fsp=3)
+        mysql = sqlalchemy.dialects.mysql.DATETIME(timezone=True, fsp=6)
```

Comment:
It will break part of alert activations logic [here](https://github.com/mlrun/mlrun/blob/9027287235ea58f0187a362d3b4718b50abe4795/server/py/framework/utils/notifications/notification_pusher.py#L192-L195) as we convert to mysql format with fsp=3. Also, higher precision will increase the size of data which is super crucial for alert activations. So at least in alert activations, I'd not change that